### PR TITLE
Create ocean-and-coastal-research.csl

### PR DESCRIPTION
**Pull request by azguth**
Reference style for use in the Ocean and Coastal Research Journal - http://www.ocr-journal.org/

### CODE REVIEW

File: ocean-and-coastal-research.csl [hunk 1]
```diff
@@ -0,0 +1,175 @@
+
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-GB">
+  <!-- This style was edited with the Visual CSL Editor (https://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>Ocean and Coastal Research</title>
+    <title-short>OCR </title-short>
+    <id>http://www.zotero.org/styles/ocean-and-coastal-research</id>
+    <link href="http://www.zotero.org/styles/ocean-and-coastal-research" rel="self"/>
+    <link href="http://www.ocr-journal.org/instructions" rel="self"/>
```
Comment (rmzelle) [inline ocean-and-coastal-research.csl:9]: @azguth, thanks. Is http://www.ocr-journal.org/ the correct website? It doesn't seem to be active. I assume we can change this to http://www.io.usp.br/index.php/ocean-coast-res/normas-e-instrucoes.html ?

**Comment by azguth** [comment]
Hi @rmzelle.
Yes, that is correct. Our new website is not up yet. I hope it will be
online anytime this week (if everything runs well)
 But you could refer to the soon-to-be old website.
Should we wait or should we use
http://www.io.usp.br/index.php/ocean-coast-res/normas-e-instrucoes.html
now and change it when the new website is online?
Thank you

**Comment by rmzelle** [comment]
Will the old URL keep working for a while, or are you migrating over soon?

If you're affiliated with the journal, I was also wondering if you could change your author instructions to provide examples of what the style format should look like. http://www.io.usp.br/images/publicacoes/BJO/Normas_OCR_-_Ingles_1.pdf currently reads:

> Ocean and Coastal Research follows the Harvard style of literature referencing. These guidelines are embedded in bibliography software such as Zotero and Mendeley, and can be found in several online sources, including http://www.citethisforme.com/harvard-referencing.

However, there are many, many "Harvard" style variants (we have over a hundred in the repository), as it really only means that the style uses an author-date format like "(Doe et al., 2020)".

**Comment by azguth** [comment]
We will migrate soon.

The new website Will have examples of the style and instructions for the
authors updated. It just needs to migrate to the institute's server.

File: ocean-and-coastal-research.csl (hunk 1 above)
Comment (azguth) [reply ocean-and-coastal-research.csl]: our site is now online! http://www.ocr-journal.org/index.php/instructions-for-authors

**Comment by rmzelle** [comment]
@azguth, thanks!

**Comment by azguth** [comment]
@rmzelle Thank you for all the help and patience.

**Pull request merged**